### PR TITLE
Replace recharts with Chartkick

### PR DESCRIPTION
**Pull request by gabek**
I kept seeing issues drawing the hardware multi-series graph.  So I swapped out the graphing library and this one is fine.

* Replace Recharts with Chartkick.
* Remove all the d3 utils stuff needed for Recharts.
* Remove the chart resize logic tied to window size since it's no longer needed.

### CODE REVIEW

**Comment by gabek** [comment]
Closes https://github.com/owncast/owncast/issues/393

File: pages/components/chart.tsx [hunk 1]
```diff
@@ -28,106 +21,49 @@ interface ChartProps {
...
+  )
 }
 
 Chart.defaultProps = {
   dataCollections: [],
   data: [],
   title: '',
 };
+
+function createGraphDatasetFromObject(dataArray) {
```
Comment (gingervitis) [inline pages/components/chart.tsx:61]: you should put these functions near the top of the file, else linter complain about "using function before it's defined"

File: pages/components/chart.tsx [hunk 2]
```diff
@@ -28,102 +21,45 @@ interface ChartProps {
...
-  }
-  
-  const windowSize = useWindowSize();
-  const chartWidth = windowSize.width * .68;
-  const chartHeight = chartWidth * .333;
-
-  const timeFormatter = (tick: string) => {
-    return timeFormat("%I:%M")(new Date(tick));
-  };
+  var renderData = [];
```
Comment (gingervitis) [inline pages/components/chart.tsx:35]: `const`!

**Comment by gabek** [comment]
Closes https://github.com/owncast/owncast/issues/393.

**Pull request merged**